### PR TITLE
Allow use of existing IAM role for EC2 instance profile

### PR DESCRIPTION
**Pull request by JBarna**
## What
* Allow the user of the module to specify an existing IAM Role name for the instance profile
* This IAM role name will be used to create the instance profile that is assigned to the EC2 instances managed by Elastic Beanstalk

## why
* Currently the module creates an IAM role and a series of permissions for the role.
* It is not possible to specify what permissions to use
* It is not possible to edit the permissions that are created
* This limitation severely limits the capability of the EC2 instances if they require other permissions to operate.

## references
* This PR will close issue 70: closes#70
* https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment/issues/70

## Additional information
I was unable to build the readme.md. I ran 
```
make init
make readme/deps
make readme
```
and received this error
```
$ make readme
Package terraform-docs already installed
make: gomplate: No such file or directory
make: *** [readme/build] Error 1
```

Please advise how to fix those errors. 

Best,
Joel

### CODE REVIEW

**Review by maximmi — DISMISSED** [review]
@JBarna please, address the comments

File: examples/complete/versions.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12.0"
+  required_version = "0.13.4"
```
Comment (jamengual) [inline examples/complete/versions.tf:2]: we discourage using a specific version unless the feature requires a specific version so keep it as ">= 0.12.0"

**Comment by aknysh** [comment]
@JBarna  please resolve the conflicts

**Comment by aknysh** [comment]
still conflicts
@boeser-stavvy thank you for taking care of this

**Pull request closed without merging**